src/backend/core/tests/api/test_organizations.py

```
@@ -430,3 +442,428 @@ def test_api_organizations_retrieve_authenticated_no_role():
+    assert response.json() == _full(
+        manual=["b.fr", "new.fr"], candidates=["kept-auto.fr"]
+    )
+    organization.refresh_from_db()
+    assert organization.proconnect_domains == _full(
+        manual=["b.fr", "new.fr"], candidates=["kept-auto.fr"]
+    )
+
+
+def test_api_organization_proconnect_domains_change_creates_log_entry():
+    """Domain add/remove is recorded as an admin LogEntry on the org."""
+    from django.contrib.admin.models import LogEntry
```

Ruff (0.16.0)

[warning] `import` should be at the top-level of a file

(PLC0415)